src/charm.py

```diff
@@ -198,6 +198,13 @@ def service_extra_args(self, service_name, config_key) -> str:
         args = self.cis_benchmark.craft_extra_args(service_name, extra_args)
         return " ".join(f"{k}={v}" for k, v in args.items())
 
+    def allow_privileged(self) -> bool:
```

Comment:
This is only a suggestion, can we make this a `@property`? Or at least change the name to `get_allow_privileged`
```suggestion
    def allow_privileged(self) -> bool:
```

Reply:
Done.  GREAT suggestion